applications/asset_tracker_v2/doc/modem_module.rst

```diff
@@ -79,7 +79,7 @@ Neighbor cell measurements
 ==========================
 
 Neighbor cell measurements can be requested by the application by sending an ``APP_EVT_DATA_GET`` event where ``APP_DATA_NEIGHBOR_CELLS`` is part of the requested data list.
-Upon reception of that event, the modem module uses the LTE link controller to start a neighbor cell  search of type :c:enum:`LTE_LC_NEIGHBOR_SEARCH_TYPE_DEFAULT`.
+Upon reception of that event, the modem module uses the LTE link controller to start a neighbor cell  search of type :c:enum:`MODEM_NEIGHBOR_SEARCH_TYPE_DEFAULT`.
```

Comment:
Should MODEM_NEIGHBOR_SEARCH_TYPE_DEFAULT be :kconfig:option:\`CONFIG_MODEM_NEIGHBOR_SEARCH_TYPE_DEFAULT\`?
```suggestion
Upon reception of that event, the modem module uses the LTE link controller to start a neighbor cell search of type :c:enum:`MODEM_NEIGHBOR_SEARCH_TYPE_DEFAULT`.
```

Reply:
You're right - should be fixed now.